Files/Views/LayoutModes/GenericFileBrowser.xaml.cs

```diff
@@ -217,50 +217,18 @@ private void ViewModel_PropertyChanged(object sender, PropertyChangedEventArgs e
         private TextBox renamingTextBox;
 
         private DispatcherTimer tapDebounceTimer;
-
-        private void AllView_PreparingCellForEdit(object sender, DataGridPreparingCellForEditEventArgs e)
+        private void AllView_BeginningEdit(object sender, DataGridBeginningEditEventArgs e)
         {
             if (ParentShellPageInstance.FilesystemViewModel.WorkingDirectory.StartsWith(AppSettings.RecycleBinPath))
             {
                 // Do not rename files and folders inside the recycle bin
-                AllView.CancelEdit(); // Cancel the edit operation
-                return;
-            }
-
-            // Only cancel if this event was triggered by a tap
-            // Do not cancel when user presses F2 or context menu
-            if (e.EditingEventArgs is TappedRoutedEventArgs)
-            {
-                if (AppSettings.OpenItemsWithOneclick)
```

Comment:
This stuff is still needed: if OpenItemsWithOneclick is enabled and you click twice on the filename it starts renaming

Reply:
@gave92 - I can add this back, but it really should not happen - we should never enter begin the edit as ItemPressed should perform the navigation.

Reply:
@Don-Vito i know but i can still trigger it 100% of times :(